libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -589,23 +582,23 @@ uint32_t AP_Frsky_Telem::calc_param(void)
         _paramID = 0;
     }
     _paramID++;
-    switch(_paramID) {
-    case 1:
-        param = gcs().frame_type(); // see MAV_TYPE in Mavlink definition file common.h
-        break;
-    case 2: // was used to send the battery failsafe voltage
-    case 3: // was used to send the battery failsafe capacity in mAh
-        break;
-    case 4:
-        param = (uint32_t)roundf(_battery.pack_capacity_mah(0)); // battery pack capacity in mAh
-        break;
-    case 5:
-        param = (uint32_t)roundf(_battery.pack_capacity_mah(1)); // battery pack capacity in mAh
-        break;
+    switch (_paramID) {
+        case 1:
```

Comment:
I'm a fan of having the cases aligned with the switch.

Randy has a slight preference for indented like this.

Both are acceptable in the style guide IIUC.

Not sure it's worth the change?

Reply:
I use astylerc from the Tools/CodeStyle with the addition of "indent-switches"